content/docs/reference/reference.json

```diff
@@ -168,6 +168,52 @@
     "services": [],
     "type": "datetime"
   },
+  "ext-data-url": {
+    "id": "ext-data-url",
+    "title": "External Data URL",
+    "short_description": "Path to the external dataset",
+    "path": "/../enterprise/reference/configure#url",
+    "enterpriseOnly": true
+  },
+  "ext-data-record-type": {
+    "id": "ext-data-record-type",
+    "title": "External Data Record Type",
+    "short_description": "Overriden by the directory structure of an archive file dataset",
```

Comment:
@cmo-pomerium these short descriptions appear in the Console:
![image](https://user-images.githubusercontent.com/2349184/173630260-6bd91184-ec5e-44dc-b70c-c9b1f08b7d63.png)

The `path` value refers to anchors also added in this PR in `configure.md`